### PR TITLE
Add action related methods

### PR DESCRIPTION
**Pull request by ramiel**
Add some methods which are useful when testing.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -17,6 +17,14 @@ export default function configureStore (middlewares = []) {
           return actions
         },
 
+        getAction (type) {
```
Comment (dmitry-zaets) [inline src/index.js]: I would suggest to name the method as `getActionByType` and maybe to add method `getAction` that accepts the function as a predicate, so the usage would be like: `getActions(action => action.type === myType)`
That would give much more flexibility to developers and will allow them to implement their own comparison functions.
What do you think?

Comment (ramiel) [reply src/index.js]: I like the recommendation and I'll do the change soon.

Comment (ramiel) [reply src/index.js]: Done, have a look.

**Comment by arnaudbenard** [comment]
I appreciate the input but I don't think it is essential to the library.

**Pull request closed without merging**